packages/block-library/src/image/view.js

```diff
@@ -29,6 +30,8 @@ const { state, actions, callbacks } = store(
 	{
 		state: {
 			currentImageId: null,
+			preloadTimers: {},
```

Comment:
Not important, but this could be a `Map` instead.

```suggestion
			preloadTimers: new Map(),
```

Reply:
✅